packages/cli/package.json

```diff
@@ -9,7 +9,9 @@
 	},
 	"scripts": {
 		"pretest": "rimraf ./tests/output",
-		"test": "jest --forceExit"
+		"test": "jest --forceExit",
+		"prepack": "ncp ../../README.md README.md",
```

Comment:
I don’t think this would work, we use yarn release to publish which triggers the changeset publish. I’d make it part of that command since changesets seems to circumvent package lifecycles. An example can be found [here](https://github.com/JoviDeCroock/prefresh/blob/main/package.json#L8)

Reply:
Ah, gotcha, thanks. Should be fixed now.